nss-day-cohort-51/nss-day-cohort-51.github.io
# Mw fix favorite developer dev card sections 146

# Description
#146 
Favorite is working!
The Favorite Developer input was removed from the modal.

## Type of change


# Testing Instructions
- Pull down the branch and serve website using npm start
- Open browser Devtools/Inspector, go to Application tab, and go to **Storage** -> Local Storage
- On a DevCard, click a Favorite button on any student, and the student's ID should populate in Local Storage. Click multiple if you like to see multiple IDs populate in the array
- 

ADDITION:
Added a "More Details" button on the Event Card.
Added styling to fall in line with existing buttons.

# Checklist:
